src/components/FileRepo/FileRepo.js

```diff
@@ -153,6 +173,7 @@ const FileRepo = compose(injectState, withTheme, withApi)(
                             <Table
                               {...props}
                               {...url}
+                              customHeaderContent={generateHeaderContent(props, selectionSQON)}
```

Comment:
if `generateHeaderContent` is made `TableHeaderContent`, this can be:
```
customHeaderContent={<TableHeaderContent {...props} sqon={selectionSQON} disabled={false} />}
```